charts/opentelemetry-operator/values.yaml

```diff
@@ -62,9 +62,27 @@ manager:
     # additional labels for the PrometheusRule
     extraLabels: {}
 
+  targetAllocatorImage:
+    repository: ""
+    tag: ""
+
+  autoInstrumentationImage:
+    java:
+      repository: ""
+      tag: ""
+    nodejs:
+      repository: ""
+      tag: ""
+    python:
+      repository: ""
+      tag: ""
+    dotnet:
+      repository: ""
+      tag: ""
```

Comment:
Let's put it closer to the `collectorImage` config? Does specifying `autoInstrumentationImage` _enable_ the auto instrumentation or needed just to override default images?

Reply:
Moved to under `collectorImage` config, and it should just be overriding the default images